photon-core/src/main/java/org/photonvision/vision/processes/PipelineManager.java

```diff
@@ -528,10 +532,6 @@ public void changePipelineType(int newType) {
 
         logger.info("Adding new pipe of type " + type + " at idx " + idx);
 
-        // type gets overritten by reflction hackery above
-        newSettings.pipelineIndex = idx;
-        newSettings.pipelineType = type;
```

Comment:
why did these get deleted?

Reply:
The type was deleted since it is no longer overwritten, the other was... a skill issue...